packages/storybook/src/theme-builder/FontFamily.stories.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable react/prop-types */
+import { Meta, StoryObj } from '@storybook/react';
+import purmerendScrapedTokens from './project-wallace-purmerend.tokens.json';
+import { getTokenValue, isDesignToken } from '@nl-design-system-unstable/tokens-lib/src/design-tokens';
+// import {} from '@utrecht/component-library-react/dist/css-module';
+import { ComboboxStory } from './Combobox';
+import { useId } from 'react';
+
+const projectWallaceWorkaround = (projectWallaceValue) => {
+  let value = projectWallaceValue;
+
+  // Project Wallace workaround
+  if (Array.isArray(value) && typeof value[0] === 'string') {
```

Comment:
Unquoten is iets dat Wallace ook nog wel zou kunnen doen, denk ik. Dat fontFamily een array is, is overigens wel volgens spec https://tr.designtokens.org/format/#font-family (mag een single string of array of strings zijn).